beets/util/__init__.py

```diff
@@ -737,10 +737,29 @@ def cpu_count():
         return 1
 
 
+def convert_command_args(args):
+    """Convert command arguments to bytes on Python 2 and strings on
+    Python 3."""
+    encoding = arg_encoding()
+    for i, elem in enumerate(args):
+        if six.PY2:
+            if isinstance(elem, six.text_type):
+                args[i] = elem.encode(encoding)
```

Comment:
Modifying an input array in place is somewhat unexpected. This function should probably return a new array (or be implemented as a generator, with `yield`).